nifi-test/pom.xml

```diff
@@ -0,0 +1,145 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>nifi</artifactId>
+        <groupId>org.apache.nifi</groupId>
+        <version>1.8.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+
```

Comment:
I don't think this should be thingbiganalytics

Reply:
Agreed. AFAIK, things like this need to be refactored, where possible, to `org.apache.nifi.*`

Reply:
Yes, this was incorrect: I'll fix in my fork and create a new pull request later on.